addons/web/static/src/core/confirmation_dialog/confirmation_dialog.js

```diff
@@ -15,11 +15,15 @@ Think twice before you click that 'Delete' button!`
 
 export class ConfirmationDialog extends Component {
     setup() {
-        this.env.dialogData.dismiss = () => this._cancel();
+        this.env.dialogData.dismiss = () => this._dismiss();
         this.modalRef = useChildRef();
         this.isProcess = false;
     }
 
+    async _dismiss() {
```

Comment:
Would move the method after "_cancel" and "_confirm" to keep alphabetical order.